src/app/api/routes/accesses.py

```diff
@@ -1,44 +1,15 @@
-from typing import List, Dict, Any
-from fastapi import APIRouter, Path, HTTPException
-from app.api import crud, security
+"""Access route read-only for debugging purpose, aims to be removed"""
+
+from typing import List
+from fastapi import APIRouter, Path
+from app.api import crud
 from app.db import accesses
-from app.api.schemas import AccessBase, AccessRead, AccessAuth, AccessCreation, Cred, CredHash
+from app.api.schemas import AccessRead
 
 
 router = APIRouter()
 
 
-async def post_access(login: str, password: str, scopes: str) -> AccessRead:
-    # Check that the login does not already exist
-    if await crud.fetch_one(accesses, [('login', login)]) is not None:
-        raise HTTPException(
-            status_code=400,
-            detail=f"An entry with login='{login}' already exists.",
-        )
-    # Hash the password
-    pwd = await security.hash_password(password)
-    access = AccessCreation(login=login, hashed_password=pwd, scopes=scopes)
-    entry = await crud.create_entry(accesses, access)
-
-    return AccessRead(**entry)
-
-
-async def update_access_pwd(payload: Cred, entry_id: int = Path(..., gt=0)) -> Dict[str, Any]:
-    entry = await crud.get_entry(accesses, entry_id)
-    # Hash the password
-    pwd = await security.hash_password(payload.password)
-    # Update the password
-    payload = CredHash(hashed_password=pwd)
-    await crud.update_entry(accesses, payload, entry_id)
-    # Return non-sensitive information
-    return {"login": entry["login"]}
-
-
-@router.post("/", response_model=AccessRead, status_code=201)
-async def create_access(payload: AccessAuth):
-    return await post_access(payload.login, payload.password, payload.scopes)
-
-
 @router.get("/{access_id}/", response_model=AccessRead)
```

Comment:
If we want to remove the access routes (it is fine by me if we think we don't need it anymore for debugging purpose), why leave those routes?

Reply:
I think he mentioned that we don't want to expose creation/modification of accesses but reading is fine

Reply:
Yes reading is fine while it does not break database consistency